ui/src/entities/Organizations.ts

```diff
@@ -7,6 +7,7 @@ export interface Organization {
   name: string;
   banner: string;
   square_logo: string;
+  employees_aggregate: any;
```

Comment:
We should avoid using `any` as much as we can. If we make this property `employee_count` instead, we get a cleaner abstraction and a definite type (`number`).